core/modules/taxonomy/taxonomy.api.php

```diff
@@ -272,6 +272,24 @@ function hook_taxonomy_term_view_alter(&$build) {
   $build['#post_render'][] = 'my_module_taxonomy_term_post_render';
 }
 
+/**
+ * Alter the results of taxonomy_term_page()
+ *
+ * This hook is called after the node listing for a taxonomy term page has
+ * been assembled in a structured array and may be used for altering that
+ * listing.
```

Comment:
Line lengths should be as close to 80 characters as possible (https://docs.backdropcms.org/doc-standards#general):

```suggestion
 * This hook is called after the node listing for a taxonomy term page has been
 * assembled in a structured array and may be used for altering that listing.
```